package.json

```diff
@@ -18,8 +17,9 @@
     "deploy:sepolia": "bash -c 'source .env && forge script DeploySepolia --rpc-url $SEPOLIA_RPC --broadcast --private-key $SEPOLIA_DEPLOYER_PK --verify --etherscan-api-key $ETHERSCAN_API_KEY'",
     "lint:check": "yarn lint:sol-tests && yarn lint:sol-logic && forge fmt --check",
     "lint:fix": "sort-package-json && forge fmt && yarn lint:sol-tests --fix && yarn lint:sol-logic --fix",
```

Comment:
What do you think about executing `yarn lint:natspec` within `yarn lint:fix`?

Reply:
@0xJabberwock that's a good idea. Perhaps it would be best to do it after introducing some sort of compact output to the natspec checker, because currently it prints a lot of logs in case natspec is missing and that may be annoying or distracting.